packages/grafana-ui/src/components/PageLayout/PageToolbar.tsx

```diff
@@ -44,7 +44,7 @@ export const PageToolbar: FC<Props> = React.memo(
     );
 
     return (
-      <div className={mainStyle}>
+      <nav className={mainStyle} aria-label="Explore toolbar">
```

Comment:
This would be a nice a11y improvement, but this component is in grafana-ui, i don't think we should be hardcoding the label here. what about passing it as a prop?